specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2023-10-01/examples/ManagedNetwork/listRule.json

```diff
@@ -0,0 +1,39 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "test-rg",
+    "workspaceName": "aml-workspace-name",
+    "api-version": "2023-10-01"
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "value": [
+          {
+            "id": "id_of_the_fqdn_rule",
```

Comment:
id property should be the arm resource ids

Reply:
Updated